tools/MultiPEN/Rscript-STRINGdb-network.xml

```diff
@@ -1,17 +1,18 @@
-<tool id="MultiPEN-StringDBNetwork" name="StringDB Network" version="0.0.3">
-    <description> Compiles network from list of genes</description>
+<tool id="STRINGdb-network" name="STRINGdb-network" version="0.0.3">
```

Comment:
Can we do the IDs all lower-case?
The name could be `STRINGdb network` this can be free text no need to the dash.

Thanks @PerlaTroncosoRey!